src/components/NavBar/NavBar.test.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import NavBar from '.';
+
+describe('NavBar', () => {
+  beforeEach(() => {
+    render(<NavBar />);
+  });
+
+  it('renders NavBar and its elements', () => {
```

Comment:
👌🏽  nice, it would be even better if you test user interaction for the navbar to open/close